### PR TITLE
Always required new variables

### PR DESCRIPTION
**Pull request by ashcrow**
Related to https://bugzilla.redhat.com/show_bug.cgi?id=1451023

### CODE REVIEW

**Comment by ashcrow** [comment]
Hrm, I would have thought https://github.com/openshift/aos-cd-jobs/pull/616 would cover this. Looking to see why it's being caught up.

**Comment by ashcrow** [comment]
Found the mismatch. The integration testing for preflight steps runs with no inventory so, of course, the variables don't end up being set causing a failure:

**File**: test/integration/openshift_health_checker/common.go
```golang
// Run runs the PlaybookTest.
func (p PlaybookTest) Run(t *testing.T) {
        // A PlaybookTest is intended to be run in parallel with other tests.
        t.Parallel()

        cmd := exec.Command("ansible-playbook", "-i", "/dev/null", p.Path)
        cmd.Env = append(os.Environ(), "ANSIBLE_FORCE_COLOR=1")
        b, err := cmd.CombinedOutput()

```

Looking to see if these can be run with an inventory or if maybe I should try a different approach.

File: inventory/byo/hosts.origin.example [hunk 1]
```diff
@@ -613,7 +613,10 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # WORKAROUND : If you must use an overlapping subnet, you can configure a non conflicting
 # docker0 CIDR range by adding '--bip=192.168.2.1/24' to DOCKER_NETWORK_OPTIONS
 # environment variable located in /etc/sysconfig/docker-network.
-# When upgrading these must be specificed!
+# When upgrading or scaling up the following must match whats in your master config!
```
Comment (michaelgugino) [inline inventory/byo/hosts.origin.example]: It's not clear that these are now mandatory and install will fail without them.

Comment (ashcrow) [reply inventory/byo/hosts.origin.example]: I'll clarify.

**Comment by ashcrow** [comment]
CI system is throwing 502's. Waiting to test again.

**Comment by ashcrow** [comment]
Infra flake
```
SYNC OPENSHIFT-ANSIBLE PULL REQUEST 
```

**Comment by ashcrow** [comment]
@sdodson PTAL

**Comment by ashcrow** [comment]
[merge]

**Pull request merged**